src/components/Help/Documentation/index.tsx

```diff
@@ -0,0 +1,217 @@
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    <span
+                      className={f('icon', 'icon-common')}
+                      data-icon="&#xf09b;"
+                    />{' '}
+                    General API documentation
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="//www.ebi.ac.uk/interpro/api/static_files/swagger/"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Swagger API documentation
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    to={{
+                      description: {
+                        main: { key: 'result' },
+                        result: { type: 'download' },
+                      },
+                      hash: '/entry/interpro/|json',
+                    }}
+                  >
+                    Code snippet generator
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="//www.ebi.ac.uk/data-protection/privacy-notice/embl-ebi-public-website"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Privacy and security
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="//www.ebi.ac.uk/ebisearch/documentation"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    EBI Search documentation
+                  </Link>
+                </li>
+              </ul>
+            </div>
+            <div className={f('flex-box')}>
+              <span
+                className={f('icon', 'icon-common', 'xl')}
+                data-icon="&#xf10d;"
+              />{' '}
+              <h5>Publications</h5>
+              <ul>
+                <li>
+                  <Link
+                    href={`${config.root.readthedocs.href}citing.html`}
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Citing InterPro
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="https://pfam-docs.readthedocs.io/en/latest/citing-pfam.html"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Citing Pfam
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div className={f('vf-grid__col--span-1')}>
+          <div className={f('box-add', 'margin-top-xxlarge')}>
+            <h3 className={f('light')}>Contact us/Additional help</h3>
```

Comment:
```suggestion
            <h3 className={f('light')}>Contact us</h3>
```

Text is too long for the sidebar.